salt/cli/daemons.py

```diff
@@ -420,29 +452,21 @@ def prepare(self):
                     permissive=self.config['permissive_pki_access'],
                     pki_dir=self.config['pki_dir'],
                 )
-                if 'proxy_log' in self.config:
-                    logfile = self.config['proxy_log']
-                else:
-                    logfile = self.config['log_file']
-                if logfile is not None and not logfile.startswith(('tcp://',
-                                                                   'udp://',
-                                                                   'file://')):
+
+                logfile = self.config.get('proxy_log') or self.config['log_file']
+                if logfile is not None and not logfile.startswith(('tcp://', 'udp://', 'file://')):
                     # Logfile is not using Syslog, verify
                     current_umask = os.umask(0o027)
                     verify_files([logfile], self.config['user'])
                     os.umask(current_umask)
 
-        except OSError as err:
-            log.exception('Failed to prepare salt environment')
-            self.shutdown(err.errno)
+        except OSError as error:
+            self.environment_failure(error)
 
         self.setup_logfile_logger()
         verify_log(self.config)
-        log.info(
-            'Setting up a Salt Proxy Minion "{0}"'.format(
-                self.config['id']
-            )
-        )
+        self.action_log_info('Setting up "{0}"'.format(self.config['id']))
```

Comment:
Why remove the indication that this is a proxy minion? I believe this was intentional.